realio/assetlist.json

```diff
@@ -7,7 +7,7 @@
         "denom_units": [
           {
             "denom": "ario",
-            "exponent": 0
+            "exponent": 18
```

Comment:
both of these denom units 'ario' and 'arst' should remain as having 0 exponent, and they  are the base units. The corresponding denom units without the 'a' at the front has the 18 exponent. Please change back to 0.

Reply:
hhh yeah sorry